tools/functional-tester/etcd-tester/tester.go

```diff
@@ -115,3 +121,77 @@ func (s *Status) setCase(c int) {
 	defer s.mu.Unlock()
 	s.Case = c
 }
+
+// checkConsistency stops the cluster for a moment and get the hashes of KV storages.
+// TODO: instead of logging to stdout, record it as an event to database.
```

Comment:
i would rather keep tester as simple as possible. probably we can have another process that can get info from tester and do the database operations.